app/editor/src/features/admin/report-templates/constants/defaultAVOverviewRazorTemplate.ts

```diff
@@ -0,0 +1,69 @@
+export const defaultAVOverviewSubjectRazorTemplate = `@inherits RazorEngineCore.RazorEngineTemplateBase<TNO.Services.Reporting.Models.ReportEngineAVOverviewModel>
+@using TNO.TemplateEngine
+Daily Today's News Online Media Overview - @($"{Instance.PublishedOn:dddd, MMMM dd, yyyy}")`;
+
+export const defaultAVOverviewBodyRazorTemplate = `@inherits RazorEngineCore.RazorEngineTemplateBase<TNO.Services.Reporting.Models.ReportEngineAVOverviewModel>
+@using System
+@using System.Linq
+@using TNO.Entities
+
+<h2 id="top">Daily Today's News Online Media Overview</h2>
+<h3>@($"{Instance.PublishedOn:dddd, MMMM dd, yyyy}")</h3>
+<br/>
+@if (Instance.Sections.Count() == 0)
+{
+  <p>There is no content in this report.</p>
+}
+@foreach (var section in Instance.Sections)
+{
+  var itemIndex = 0;
+  var itemCount = section.Items.Count(i => i.ItemType == AVOverviewItemType.Story);
+  <div style="padding-bottom: 1rem;">
+    <div style="background-color: black; color: white; text-align: center; font-weight: bold;">
```

Comment:
Is the reason that there is so much inline css because the templates cannot use stylesheets?

Just wondering if there's a way we could come up with a set of predefined styles that can be used by template creators here?

Reply:
These are emails.  They say the `<style>` tag is supported by most email viewers.  We could probably use it, but for the basic styling required currently I opted for simplicity.

Reply:
I've extracted the styling.  I'm not certain it actually makes it better in this specific email use-case, but generally I think it's wise to do this.